platforms/core-configuration/configuration-problems-base/src/main/kotlin/org/gradle/internal/cc/impl/problems/JsonWriter.kt

```diff
@@ -29,6 +29,13 @@ class JsonWriter(private val writer: Writer) {
         endObject()
     }
 
+    private fun jsonListItem(body: () -> Unit) {
```

Comment:
💅 Keep the `private` modifier placement consistent across the file

Reply:
Fixed